### PR TITLE
fix: SLA param changes that push an LP out of meeting commitment rese…

### LINKED ISSUE #11065: [Bug]: `currentEpochFractionOfTimeOnBook` reporting value bigger than `1`
### Problem encountered

Since the proposal to change some SLA settings on some mainnet markets was enacted, the `currentEpochFractionOfTimeOnBook` for some LP's are being reported as being bigger than one:
[https://vega-data.nodes.guru:3008/api/v2/liquidity/providers?marketId=4e9081e20e9e81f3e74[…]27bd526b46f1db2db315df08637331beaffa8ca4fd2c7065a4d7e4a7](https://vega-data.nodes.guru:3008/api/v2/liquidity/providers?marketId=4e9081e20e9e81f3e747d42cb0c9b8826454df01899e6027a22e771e19cc79fc&partyId=6d4c7e9727bd526b46f1db2db315df08637331beaffa8ca4fd2c7065a4d7e4a7)

This makes no sense since how can they be meeting their commitment more than 100% of the time.

We need to work out why this has happened, and to understand the impact of rolling into the next epoch with a value of `currentEpochFractionOfTimeOnBook` bigger than one. Will it panic? Will it overflow and cause havoc?
```
// p = (1-[timeBookFraction-commitmentMinTimeFraction/1-commitmentMinTimeFraction]) * slaCompetitionFactor
	return one.Sub(
		timeBookFraction.Sub(e.slaParams.CommitmentMinTimeFraction).Div(one.Sub(e.slaParams.CommitmentMinTimeFraction)),
	).Mul(e.slaParams.SlaCompetitionFactor)
```

### Observed behaviour

SLA `currentEpochFractionOfTimeOnBook` seen as being bigger than 1.

### Expected behaviour

`currentEpochFractionOfTimeOnBook` should never be bigger than one.

### Steps to reproduce

```bash
1. Submit `xyz` transaction
2. With this payload
3. Query `abc` API
4. Observe the error
```

### Software version

v0.74.10

### Failing test

_No response_

### Jenkins run

_No response_

### Configuration used

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by wwestgarth**
closes #11065

### CODE REVIEW

File: core/liquidity/v2/sla.go [hunk 1]
```diff
@@ -88,7 +89,7 @@ func (e *Engine) calculateCurrentTimeBookFraction(now, start time.Time, s time.D
 		timeBookFraction = num.DecimalFromInt64(s.Nanoseconds()).Div(num.DecimalFromInt64(lNano))
 	}
 
-	return timeBookFraction
+	return num.MinD(num.DecimalOne(), timeBookFraction)
```
Comment (EVODelavega) [inline core/liquidity/v2/sla.go:92]: are we sure we want to keep this? If the value > 1, wouldn't that imply that the party hasn't met the commitment last time around, and therefore the fraction would _have_ to be < 1

Comment (wwestgarth) [reply core/liquidity/v2/sla.go:92]: I'm not sure I follow. It should always be that `timeBookFraction <= 1` and if its not something has gone wrong so the cap is just there for safety.

**Review by EVODelavega — DISMISSED** [review]

**Review by guoguojin — APPROVED** [review]

File: core/liquidity/v2/sla.go [hunk 2]
```diff
@@ -87,8 +88,7 @@ func (e *Engine) calculateCurrentTimeBookFraction(now, start time.Time, s time.D
 	if lNano > 0 {
 		timeBookFraction = num.DecimalFromInt64(s.Nanoseconds()).Div(num.DecimalFromInt64(lNano))
 	}
-
-	return timeBookFraction
+	return num.MinD(num.DecimalOne(), timeBookFraction)
```
Comment (davidsiska-vega) [inline core/liquidity/v2/sla.go:91]: Can we also cap from below by `0.0`?

**Review by davidsiska-vega — DISMISSED** [review]

**Comment by EVODelavega** [comment]
One failed system test, looks like a rare/random failure, re-running the test itself: https://jenkins.vega.rocks/job/common/job/system-tests-wrapper/53752

File: core/liquidity/v2/sla.go (hunk 2 above)
Comment (EVODelavega) [reply core/liquidity/v2/sla.go:91]: negative values here necessitate time-travel, which is currently unsupported by vega. :)

**Pull request merged**